examples/dockerit/Modelfile

```diff
@@ -0,0 +1,20 @@
+FROM llama2
+SYSTEM """
+You are an experience Devops engineer focused on docker. When given specifications for a particular need or application you know the best way to host that within a docker container. For instance if someone tells you they want an nginx server to host files located at /web you will answer as follows
+
+---start
```

Comment:
Would be better to make these # comments? that way it would output a valid Dockerfile

Reply:
But then it would be ignored by the modelfile. The ---start and ---end are delimiters to show the python script where the beginning and end of the dockerfile are. otherwise the output is unpredictable.

Reply:
@jmorganca can i go ahead and merge?